### PR TITLE
Fixes Instrument APM + Adds what's new 

### PR DESCRIPTION
**Pull request by bmorelli25**
## Summary

* Fixes the broken cross-doc includes in the instrument-apps section
* Adds "what's new" placeholder content

## Related

Closes https://github.com/elastic/observability-docs/issues/186
Closes https://github.com/elastic/observability-docs/issues/179 (Tracking for content will be handled in the new issue)

### CODE REVIEW

File: docs/en/observability/whats-new.asciidoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+coming[8.0.0]
+
+// Other versions:
+// {observability-guide-all}/7.10/whats-new[7.10] |
+// https://www.elastic.co/blog/whats-new-elastic-observability-7-9-0-unified-agent-kpi-overview-dashboard[7.9] |
+// https://www.elastic.co/blog/elastic-observability-7-8-0-released[7.8] |
+// https://www.elastic.co/blog/elastic-observability-7-7-0-released[7.7] |
+// https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
+// https://www.elastic.co/blog/elastic-observability-7-5-0-released[7.5] |
+// https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:15]: I think these links to other releases are helpful, even if they do link to blogs. I could only find release blogs back to 7.4.

**Review by EamonnTP — APPROVED** [review]

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
I'm not sure why this is failing -- it worked locally.

```
08:57:19 INFO:build_docs:asciidoctor: ERROR: instrument-apps.asciidoc: line 22: include file not found: /tmp/docsbuild/qaSsMmfBdn/apm-server/docs/tab-widgets/install-agents-widget.asciidoc
08:57:19 INFO:build_docs:asciidoctor: ERROR: instrument-apps.asciidoc: line 36: include file not found: /tmp/docsbuild/qaSsMmfBdn/apm-server/docs/tab-widgets/configure-agent-widget.asciidoc
08:57:19 INFO:build_docs:asciidoctor: ERROR: instrument-apps.asciidoc: line 43: include file not found: /tmp/docsbuild/qaSsMmfBdn/apm-server/docs/tab-widgets/configure-server-widget.asciidoc
```

**Comment by lcawl** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Pull request merged**